dashboard/modules/job/job_manager.py

```diff
@@ -217,8 +224,16 @@ async def run(
                     self._status_client.put_status(self._job_id,
                                                    JobStatus.SUCCEEDED)
                 else:
-                    self._status_client.put_status(self._job_id,
-                                                   JobStatus.FAILED)
+                    log_tail = self._log_client.tail_logs(self._job_id)
+                    if log_tail:
```

Comment:
log_tail is worst case an empty string so we won't hit the else case i think ? also might be more explicit by using string comparison conditionals here rather than plain if

Reply:
ah yeah empty string returns False here, but you're right let me do it explicitly